pierce-portfolio/portfolio/src/main/webapp/fotos.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Can you fix the indentation of all the HTML file base on the HTML style guide?

Reply:
Yes